generators/generator-botbuilder/generators/app/templates/core/index.js

```diff
@@ -14,7 +14,15 @@ const restify = require('restify');
 
 // Import required bot services.
 // See https://aka.ms/bot-services to learn more about the different parts of a bot.
-const { BotFrameworkAdapter, ConversationState, InputHints, MemoryStorage, UserState } = require('botbuilder');
+const {
+    createBotFrameworkAuthenticationFromConfiguration,
+    CloudAdapter,
+    ConfigurationServiceClientCredentialFactory,
+    ConversationState,
+    InputHints,
+    MemoryStorage,
+    UserState
+} = require('botbuilder');
```

Comment:
We could sort the imports like the samples bots

Reply:
Done. Thanks!